ksp_plugin/flight_plan_optimizer.cpp

```diff
@@ -564,6 +674,90 @@ EvaluateGateauxDerivativeOfDistanceToCelestialWithReplacement(
   return (distance_δh - distance) / h;
 }
 
+Angle FlightPlanOptimizer::EvaluateRelativeInclinationlWithReplacement(
+    Celestial const& celestial,
+    NavigationFrame const& frame,
+    Angle const& target_inclination,
+    HomogeneousArgument const& homogeneous_argument,
+    NavigationManœuvre const& manœuvre,
+    int index) {
+  auto const [time, barycentric_degrees_of_freedom] =
+      EvaluatePeriapsisWithReplacement(
+          celestial, homogeneous_argument, manœuvre, index);
+  auto const navigation_degrees_of_freedom =
+      frame.ToThisFrameAtTime(time)(barycentric_degrees_of_freedom);
+  auto const r = navigation_degrees_of_freedom.position() - Navigation::origin;
+  auto const v = navigation_degrees_of_freedom.velocity();
+  Angle const i =
+      AngleBetween(Wedge(r, v), Bivector<double, Navigation>({0, 0, 1}));
+  Angle const signed_relative_inclination = [](Angle const α) {
```

Comment:
Why the lambda?  It makes the code hard to read.

Reply:
Filed #3768, the argument reduction situation is a mess.